scalafmt-tests/src/test/scala-2.12/org/scalafmt/ScalafmtProps.scala

```diff
@@ -33,7 +34,14 @@ class ScalafmtProps extends FormatAssertions {
           case Formatted.Success(formatted) =>
             assertFormatPreservesAst[Source](code, formatted)
             val formattedSecondTime = Scalafmt.format(formatted, config).get
-            assertNoDiff(formattedSecondTime, formatted, "Idempotency")
+            Diffs.assertNoDiff(formattedSecondTime, formatted, diff => {
+              throw DiffFailure(
+                "Idempotency",
+                formatted,
+                formattedSecondTime,
+                diff
+              )
+            }, "Idempotency")
```

Comment:
optional: should we at some point rename it to "idempotencE"? https://en.wikipedia.org/wiki/Idempotence

Reply:
done